hpOneView/resources/networking/logical_interconnects.py

```diff
@@ -456,5 +463,52 @@ def update_qos_aggregated_configuration(self, id_or_uri, qos_configuration, time
         uri = self._client.build_uri(id_or_uri) + self.QOS_AGGREGATED_CONFIGURATION
         return self._client.update(qos_configuration, uri=uri, timeout=timeout)
 
+    def update_telemetry_configurations(self, tc_id_or_uri, configuration, id_or_uri=None, timeout=-1):
+        """
+        Updates the telemetry configuration of a logical interconnect. Changes to the telemetry configuration are
+        asynchronously applied to all managed interconnects.
+
+        Args:
+            id_or_uri:
+                Can be either the logical interconnect id or uri.
+            tc_id_or_uri:
+                Can be either the telemetry configuration id or uri.
+            configuration:
+                The telemetry configuration for the logical interconnect.
+            timeout:
+                Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation in
+                OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: The telemetry configuration.
+        """
+        uri = self.__build_subresource_uri(id_or_uri, tc_id_or_uri, 'telemetry-configurations')
+        return self._client.update(configuration, uri=uri, timeout=timeout,
+                                   default_values=self.SETTINGS_TELEMETRY_CONFIG_DEFAULT_VALUES)
+
+    def get_ethernet_settings(self, id_or_uri):
+        """
+        Gets the Ethernet interconnect settings for the Logical Interconnect.
+
+        Args:
+            id_or_uri:
+                Can be either the logical interconnect id or the logical interconnect uri.
```

Comment:
Just to keep the standard
 -> Can be either the logical interconnect id or uri.